### PR TITLE
Ft 280 url sync provider gets duplicated on edit

### PR DESCRIPTION
**Pull request by MightyDevKnight**

### CODE REVIEW

File: src/app/components/Initiator.tsx [hunk 1]
```diff
@@ -186,7 +194,7 @@ export function Initiator() {
             break;
           }
           case MessageFromPluginTypes.API_PROVIDERS: {
-            dispatch.uiState.setAPIProviders(pluginMessage.providers);
+            dispatch.uiState.setAPIProviders(removeDuplicatedProviders(pluginMessage.providers));
```
Comment (six7) [inline src/app/components/Initiator.tsx:197]: i dont think we need to remove duplicated providers. we should rather not duplicate them in the first place.

Comment (MightyDevKnight) [reply src/app/components/Initiator.tsx:197]: You assigned this to @Hiroshi, right?

Comment (six7) [reply src/app/components/Initiator.tsx:197]: Oh sorry, meant you. Was late for me 😅

File: src/app/components/StorageItemForm/URLForm.tsx [hunk 2]
```diff
@@ -40,6 +47,22 @@ export default function URLForm({
     }
   }, [values, onSubmit]);
 
+  useEffect(() => {
+    if (!isNew) {
```
Comment (LiamMartens) [inline src/app/components/StorageItemForm/URLForm.tsx:51]: This logic seems a bit weird? What is the intention here? You are removing the old provider if the one you are editing is not new? Should we not just fix the issue where the actual provider is not updated? This feels like a monkey patch.

**Review by LiamMartens — CHANGES_REQUESTED** [review]

**Comment by six7** [comment]
Can you merge `next` in?

**Pull request closed without merging**